src/MSBuild/Resources/xlf/Strings.de.xlf

```diff
@@ -126,16 +126,16 @@ Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.
 
                      This flag is experimental and may not work as intended.
     </source>
-        <target state="new">  -isolateProjects[:True|False]
-                     Causes MSBuild to build each project in isolation.
+        <target state="translated">  -isolateProjects[:True|False]
+                     Führt dazu, dass MSBuild jedes Projekt isoliert erstellt.
 
-                     This is a more restrictive mode of MSBuild as it requires
-                     that the project graph be statically discoverable at
-                     evaluation time, but can improve scheduling and reduce
-                     memory overhead when building a large set of projects.
-                     (Short form: -isolate)
+                     Dies ist ein restriktiverer Modus von MSBuild, da er erfordert, dass
+                     das Projektdiagramm zur Auswertungszeit statisch ermittelbar ist. Er kann jedoch 
```

Comment:
`Projectdiagramm` is a weird translation for `project graph`, we're not talking about a literal drawing here but the abstract concept of a graph as in https://en.wikipedia.org/wiki/Graph_(discrete_mathematics).

I think we can use `Projektgraph` instead since the word has the same meaning in German: https://de.wikipedia.org/wiki/Graph_(Graphentheorie)